.github/workflows/repository.yml

```
@@ -4,24 +4,22 @@ jobs:
 
   tests:
     name: Tests
-    container: 2factorauth/twofactorauth:latest
     runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-latest
    env: 
      BUNDLE_WITHOUT: jekyll_plugins
```

https://github.com/ruby/setup-ruby#bundle-config